tests/Functional/InstallWithSymfonyTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Spatie\SymfonyIgnitionBundle\Tests\Functional;
+
+use Symfony\Component\DomCrawler\Crawler;
+use Symfony\Component\Process\Process;
+
+class InstallWithSymfony extends FunctionalTest
+{
+    /**
+     * Provide Symfony versions to test
+     *
+     * @todo Also provide PHP versions that each Symfony version supports. Will
+     *   require the test host to have Docker.
+     */
+    public function versionProvider()
+    {
+        return [
+            'Symfony 5.4' => ['5.4.*'],
+            'Symfony 6.0' => ['6.0.*'],
+        ];
+    }
```

Comment:
I think that could be put into the abstract test case `FunctionalTest` so wen we add more test classes we can just use that provider again.